cmake/Projects.cmake

```diff
@@ -173,12 +173,7 @@ if("${CMAKE_INSTALL_CONFIG_NAME}" MATCHES "^([Rr][Ee][Ll][Ee][Aa][Ss][Ee])$")
             message(STATUS "${install_output_folder}/engine.pak generated")
         endif()
     endif()
-    file(WRITE ${install_output_folder}/project.json 
-"{
-    \"project_name\": \"@project_name@\"
-}")
-endif()
-]=])
+
```

Comment:
Needs a closing `]=])` here.

Reply:
That it does.

Reply:
Actually, I think it also needs that `endif()` as well.  I'm getting Flow control error with these changes.

Reply:
Ah, you got it.

Reply:
It has been fixed now.
You also need the changes from #4454

Reply:
Right, I commented that part out for now.